(docs-rel-4-7-a) PEM-7965: Documenting GPU quotas (#7782)
spectrocloud/librarium: docs/docs-content/clusters/cluster-management/namespace-management.md

   The **Settings** pane displays with **RBAC** preselected and the **Namespaces** tab opened by default.

   ![Cluster Settings pane with the location where the namespace is listed](/clusters_cluster-management_namespace-create_4-7.webp)

5. Type a unique namespace name in the **Namespace name or Regex** field and click **Confirm** at right.

   ![Cluster Settings pane showing how to add new namespace](/clusters_cluster-management_namespace-create-new_4-7.webp)

6. You can assign resource quotas now or at a later time. To learn how, check out
   [Assign Resource Quotas](namespace-management.md#assign-resource-quotas).

   For details on how to configure RBAC for namespaces, check out the
   [RBAC and NS Support](cluster-rbac.md#palette-roles-and-kubernetes-roles) guide.

7. Once you have made all needed changes, click **Save Changes**.

### Validate

Validate that the namespace was successfully created.

1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.

2. Navigate to the left **Main Menu** and click on **Clusters**.

3. Select the cluster that contains the namespace you created and view its details.

> 4. In the **Settings** pane, click **Cluster Settings** and then select **RBAC** > **Namespaces** tab.

Reviewer: ```suggestion
4. In the **Settings** pane, click **Cluster Settings**, and then select **RBAC** > **Namespaces** tab.
```
https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1765933057/Spectro+Cloud+Internal+Style+Guide#Commas

2. Navigate to the left **Main Menu** and click on **Clusters**.

3. Select the cluster with the namespace to which you will assign cluster quotas.

4. Navigate to the **Workloads** > **Namespaces** tab, and click the **Manage Namespaces** button.

5. The **Settings** pane displays with **RBAC** > **Namespaces** preselected.

> 6. Select the three-dot menu on the namespace and select **Edit**.

Reviewer: ```suggestion
6. Click on the three-dot menu on the namespace and select **Edit**.
```

You don't really select the menu, you rather click on it.

> ![Cluster Settings pane displaying Workspace Quota section of Namespaces tab](/clusters_cluster-management_ns-resource-quota_4-7.webp)

Reviewer: The indentation of the picture is not correct, it should be moved to the right side like the previous pictures on this page. I also suspect that the picture size is not correct, it seems different from previous pictures. The resolution is usually 1615x1065.

> 7. Type the number of CPU, Memory and GPUs to allocate to the namespace, and save your changes.

Reviewer: ```suggestion
7. Specify the number of CPUs and GPUs, and the amount of memory (in GB) to allocate to the namespace. The **Confirm** your changes.
```
"number of memory" seems a bit off

> :::info

Reviewer: The info admonition indentation is not correct, it should be moved to the right and be the same as for 2 first pictures on the page.

> When using **GPU Allocation** you must use a whole number and must select the vendor from the dropdown. You must also

Reviewer: Are you sure the first sentence should be mention here? I'd probably mention the whole number on step 7. And the dropdown part is somewhat obvious from the screenshot.

Author: Removed the screenshot and will keep the written step in.

ensure that the appropriate GPU device plugin is installed and compatible with your nodes to enforce the quota as
Palette does not verify GPU vendor selection.

> At this time, NVIDIA is the only supported vendor.

Reviewer: I'd probably mention it as a limitation, not part of an info admonition.

> At this time, NVIDIA is the only supported vendor.

Reviewer: ```suggestion
Currently, NVIDIA is the only supported vendor.
```

3. Select the cluster that contains the namespace you want to delete and view its details.

> 4. In the **Settings** pane, select **Cluster Settings** and select **RBAC** > **Namespaces** tab.

Reviewer: ```suggestion
4. In the **Settings** pane, select **Cluster Settings**, and then select **RBAC** > **Namespaces** tab.
```

I think this is compatible with the [Style guide](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1765933057/Spectro+Cloud+Internal+Style+Guide#Angle-Brackets-(%3E))
